google/api_core/client_logging.py

```diff
@@ -12,10 +12,11 @@
 # TODO(https://github.com/googleapis/python-api-core/issues/761): Update this list to support additional logging fields.
 _recognized_logging_fields = [
     "httpRequest",
-    "rpcName",
+    "httpRequestMethod",
+    "httpRequestUrl" "rpcName",
```

Comment:
As per https://cloud.google.com/logging/docs/structured-logging#structured_logging_special_fields, `httpRequest` should be of type `HttpRequest`, [which](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#HttpRequest) includes the fields `requestMethod` and `requestURL` internally.


```suggestion
    "rpcName",
```